### ISSUE TITLE
Doesn't work on fish

### ISSUE THREAD

**Issue by dy-tea** [YiNNx/cmd-wrapped#39]
Running `cmd-wrapped -s fish 2024`:

![image](https://github.com/user-attachments/assets/b11e70a3-391d-4e7f-b165-bbeebf987d12)

Noticed that in the code you call fish with a `-show-time` option, which does not exist. 

Instead just call `fish -c history` which displays the history in the format:

```
2023-10-14 16:37:42 sudo pacman -S yay
YYYY-MM-DD HH:MM:SS command
```

**Comment by MemerGamer** [comment]
I have the same problem

**Comment by YiNNx** [comment]
Could you share the fish version and operating system?

**Comment by dy-tea** [comment]
`fish`: version 3.7.1
`kernel`: 6.12.1-2-cachyos

**Comment by feebrian** [comment]
hey, i think i figured it out this issue, there is a line in `history.rs` that says `history -show-time='%s;'` but the correct one is `history --show-time='%s;'` just change that then `cargo build` and you will get the history list displayed :)
![2024-12-08-064850_1920x1080_scrot](https://github.com/user-attachments/assets/12027b2e-d268-456b-a92f-8e20479f63b5)

**Comment by YiNNx** [comment]
@feebrian Thanks!

Fixed in #41

**Comment by feebrian** [comment]
> @feebrian Thanks!
> 
> Fixed in #41

yup, i got you
